corehq/toggles.py

```diff
@@ -164,8 +168,6 @@ def _get_identifier(self, item):
     def enabled(self, item, **kwargs):
         if settings.UNIT_TESTING:
             return False
-        elif item in self.always_disabled:
-            return False
```

Comment:
Removing this means that the predictably random calculation could enable this toggle for a domain that is `always_disabled` since it comes first in the `or` condition below. Is that expected/intentional?